app/src/main/java/com/elavon/converge/TransactionService.java

```diff
@@ -243,5 +271,50 @@ public void checkCard(Payment payment, String serviceCode, String cardHolderName
             //shouldn't be called - just continue
             callback.onContinue();
         }
+
+        @Override
+        public void getChildTransactions(String s, String s1, IPoyntGetTransactionsListener iPoyntGetTransactionsListener) throws RemoteException
+        {
+
+        }
+
+        @Override
+        public void checkPayment(Bundle bundle, String s, IPoyntCheckPaymentListener iPoyntCheckPaymentListener) throws RemoteException
+        {
```

Comment:
can you add:
            PoyntError error = new PoyntError();
            error.setReason("unsupported operation");
            callback.onFailure(error, requestId);

Reply:
FYI, there are existing calls here which are also unimplemented.

Reply:
I think they must've been overlooked. Yes it's a best practice to return an error as not implemented so the caller doesn't wait for ever for a callback. Can you take care of that please ?

Reply:
updated in new commit. please re-review. thanks!